### PR TITLE
Feanil/asgard functions

### PR DESCRIPTION
**Pull request by feanil**
@macdiesel the other PR got auto-closed due to git shennanigans.  But this is the PR from last week with your comments addressed.

### CODE REVIEW

File: tubular/asgard.py [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+        u'test-edx-edxapp': [
+            u'test-edx-edxapp-v007',
+        ],
+        u'test-edx-worker': [
+            u'test-edx-worker-v004',
+        ]
+    }
+
+    Raises:
+        BackendDataError: We get bad
```
Comment (macdiesel) [inline tubular/asgard.py]: Lets round out this comment.

File: tubular/tests/test_asgard.py [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+      "loadtest-edx-edxapp-v059"
+    ]
+  }
+]"""
+
+
+class TestAsgard(unittest.TestCase):
+
+    @httpretty.activate
+    def test_clusters_for_asgs(self):
```
Comment (macdiesel) [inline tubular/tests/test_asgard.py]: I think this could be written as data driven test: https://ddt.readthedocs.org/en/latest/example.html

**Comment by macdiesel** [comment]
Done with my pass, just a few nits.

**Comment by feanil** [comment]
@macdiesel any other concerns?  Am I good to merge this?

**Comment by macdiesel** [comment]
:ship: :it: :+1:

**Pull request merged**